automation.yaml

```diff
@@ -0,0 +1,5 @@
+distros:
+  - fc24
```

Comment:
For master we don't need fc24 AFAIK.

Reply:
ok n/p I just added what was in the YAML in the 'jenkins' repo